api/src/test/java/io/cloudevents/CloudEventBuilderTest.java

```diff
@@ -30,15 +31,18 @@
     public void testBuilderWithData() {
 
         // given
-        final Map<String, String> keyValueStore = Map.of("key1", "value1", "key2", "val2");
+        final Map<String, String> keyValueStore = new HashMap<>();
+        keyValueStore.put("key1", "value1");
+        keyValueStore.put("key2", "value2");
         final String eventId = UUID.randomUUID().toString();
         final URI src = URI.create("/trigger");
         final String eventType = "My.Cloud.Event.Type";
         final String eventTypeVersion = "2.0";
         final ZonedDateTime eventTime = ZonedDateTime.now();
         final String contentType = "application/json";
         final URI schemaUri = URI.create("http://cloudevents.io/schema");
-        final Map<String, String> extensionData = Map.of("foo", "bar");
+        final Map<String, String> extensionData = new HashMap<>();
+        extensionData.put("foo", "bar");
```

Comment:
:crying_cat_face:   but agreed :smile:

Reply:
The java9 solution to construct maps is amazing!

Any idea to release it?